src/Core/Context.js

```diff
@@ -154,20 +154,53 @@ ContextConsumer[SharedSitePrototype] = ObjectCreate(Component[SharedSitePrototyp
 	}
 })
 
+ContextProvider[SharedSiteDisplayName] = 'ContextProvider'
+ContextConsumer[SharedSiteDisplayName] = 'ContextConsumer'
+
 /**
  * @param {any} value
  * @return {{Provider, Consumer}}
  * @public
  */
 function createContext (value) {
+	var CustomContextProvider, Provider, Consumer
```

Comment:
Lets leave theses updates to `createContext` for another PR that also adds `contextTypes`.